app/components/widgets/forms/image-upload.js

```diff
@@ -78,6 +78,23 @@ export default Component.extend({
       if (!this.needsConfirmation) {
         this.set('selectedImage', null);
         this.set('imageUrl', null);
+        let payload = {
```

Comment:
what is this?

user.set(propertyamen, null);
user.save();

Reply:
@CosmicCoder96 I was trying to send a PATCH request to the user API but I believe this is more feasible. Updating it.